content/installation/python/PythonInstall.md

```diff
@@ -92,13 +102,6 @@ app = get_wsgi_application()
 app = ContrastMiddleware(app)
 ```
 
-## Configuration
+## Next Steps
 
-Download a standard configuration file from the Contrast application. You must place the file in the web application's *config* directory, and define the following fields, at a minimum:
-
-``` yaml
-agent:
-  service:
-    host: 
-    port:
-```
+Once the installation process is complete, you can update the agent's [configuration](installation-pythonconfig.html) file. 
```

Comment:
@jleo32 I know it goes beyond the main purpose of this PR, but I wanted to take the chance to update this piece and direct readers to the common config doc.

Reply:
Sounds good, thank you!